File: libs/adsp-service-net-sdk/Tenant.cs
using System.Text.Json.Serialization;
using Adsp.Sdk.Utils;

namespace Adsp.Sdk;
public class Tenant
{
  [JsonConverter(typeof(AdspIdJsonConverter))]
  public AdspId? Id { get; set; }
  public string? Name { get; set; }
  public string? Realm { get; set; }
  public string? AdminEmail { get; set; }
}

File: libs/adsp-service-spring-sdk/src/main/java/ca/ab/gov/alberta/adsp/sdk/tenant/Tenant.java
package ca.ab.gov.alberta.adsp.sdk.tenant;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import ca.ab.gov.alberta.adsp.sdk.AdspId;

@JsonIgnoreProperties(ignoreUnknown = true)
public final class Tenant {
  private AdspId id;
  private String name;
  private String realm;

  public AdspId getId() {
    return id;
  }

  public void setId(AdspId id) {
    this.id = id;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public String getRealm() {
    return realm;
  }

  public void setRealm(String realm) {
    this.realm = realm;
  }
}
